src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -129,6 +129,7 @@ The .NET Foundation licenses this file to you under the MIT license.
       <NativeLibrary Condition="'$(_targetArchitecture)' == 'x64'" Include="$(IlcSdkPath)$(VxSortSupportName)$(LibFileExt)" />
       <NativeLibrary Include="$(IlcSdkPath)$(StandaloneGCSupportName)$(LibFileExt)" />
       <NativeLibrary Condition="'$(LinkStandardCPlusPlusLibrary)' != 'true' and '$(StaticICULinking)' != 'true'" Include="$(IlcSdkPath)libstdc++compat.a" />
+      <NativeLibrary Condition="'$(UseSystemZlib)' != 'true'" Include="$(IlcSdkPath)libz.a" />
```

Comment:
These target files are their own universe. The get shipped in a native AOT package without the rest of the dotnet/runtime build system. You will need to duplicate the UseSystemZlib property setting here using the `_targetOS` property.

Reply:
I have addressed this with https://github.com/dotnet/runtime/pull/102403/commits/a60199a9537c1a043c6f62ce09ac58e8fadf4d66 . Does it look good to you?